types/pluralize/index.d.ts

```diff
@@ -1,79 +1,77 @@
 
-interface PluralizeStatic {
-    /**
-     * Pluralize or singularize a word based on the passed in count.
-     *
-     * @param word
-     * @param count
-     * @param inclusive
-     */
-    (word: string, count?: number, inclusive?: boolean): string;
+/**
+ * Pluralize or singularize a word based on the passed in count.
+ *
+ * @param word
+ * @param count
+ * @param inclusive
+ */
+declare function pluralize(word: string, count?: number, inclusive?: boolean): string;
 
-    /**
-     * Pluralize a word based.
-     *
-     * @param word
-     */
-    plural(word: string): string;
+declare namespace pluralize {
+  /**
+   * Pluralize a word based.
+   *
+   * @param word
+   */
+  function plural(word: string): string;
 
-    /**
-     * Singularize a word based.
-     *
-     * @param word
-     */
-    singular(word: string): string;
+  /**
+   * Singularize a word based.
+   *
+   * @param word
+   */
+  function singular(word: string): string;
 
-    /**
-     * Add a pluralization rule to the collection.
-     *
-     * @param rule
-     * @param replacement
-     */
-    addPluralRule(rule: string|RegExp, replacemant: string): void;
+  /**
+   * Add a pluralization rule to the collection.
+   *
+   * @param rule
+   * @param replacement
+   */
+  function addPluralRule(rule: string | RegExp, replacemant: string): void;
 
-    /**
-     * Add a singularization rule to the collection.
-     *
-     * @param rule
-     * @param replacement
-     */
-    addSingularRule(rule: string|RegExp, replacemant: string): void;
+  /**
+   * Add a singularization rule to the collection.
+   *
+   * @param rule
+   * @param replacement
+   */
+  function addSingularRule(rule: string | RegExp, replacemant: string): void;
 
-    /**
-     * Add an irregular word definition.
-     *
-     * @param single
-     * @param plural
-     */
-    addIrregularRule(single: string, plural: string): void;
+  /**
+   * Add an irregular word definition.
+   *
+   * @param single
+   * @param plural
+   */
+  function addIrregularRule(single: string, plural: string): void;
 
-    /**
-     * Add an uncountable word rule.
-     *
-     * @param word
-     */
-    addUncountableRule(word: string|RegExp): void;
+  /**
+   * Add an uncountable word rule.
+   *
+   * @param word
+   */
+  function addUncountableRule(word: string | RegExp): void;
 
-    /**
-     * Test if provided word is plural.
-     *
-     * @param word
-     */
-    isPlural(word: string): boolean;
+  /**
+   * Test if provided word is plural.
+   *
+   * @param word
+   */
+  function isPlural(word: string): boolean;
 
-    /**
-     * Test if provided word is singular.
-     *
-     * @param word
-     */
-    isSingular(word: string): boolean;
+  /**
+   * Test if provided word is singular.
+   *
+   * @param word
+   */
+  function isSingular(word: string): boolean;
 }
 
-declare module "pluralize" {
-    export = pluralize;
-}
-declare var pluralize: PluralizeStatic;
+export = pluralize;
```

Comment:
There should also be a `export as namespace pluralize;` declaration because this is a UMD module